src/Shared/INodePacket.cs

```diff
@@ -189,6 +189,24 @@ internal enum NodePacketType : byte
         /// Message sent back to a node informing it about the resource that were granted by the scheduler.
         /// </summary>
         ResourceResponse,
+
+        /// <summary>
+        /// Command in form of MSBuild command line for server node - MSBuild Server.
+        /// Keep this enum value constant intact as this is part of contract with dotnet CLI
+        /// </summary>
+        ServerNodeBuilCommand = 0xF0,
+
+        /// <summary>
+        /// Response from server node command
+        /// Keep this enum value constant intact as this is part of contract with dotnet CLI
+        /// </summary>
+        ServerNodeResponse = 0xF1,
```

Comment:
nit: Would it be more appropriate to call it `ServerNodeResult` or `ServerNodeBuildResult`? "Response" tends to be used as the counterpart to "Request" here.

Reply:
Make sense